addons/website/views/snippets/s_banner.xml

```diff
@@ -7,7 +7,7 @@
         <div class="container">
             <div class="row s_nb_column_fixed">
                 <div class="col-lg-7 bg-white jumbotron rounded pt32 pb32" data-name="Box">
-                    <h1 class="s_title_thin"><font style="font-size: 62px;">Sell Online. Easily.</font></h1>
```

Comment:
@sdegueldre Note: you forgot some s_title_XXX ;) Important to remove those since they still work thanks to 000.scss but will stop working at a random point where the user won't use the 000 version anymore. Note: we have to add the version number in custom template... I'll do that post-merge.